Flixel Features/SetTileProperties/source/Main.hx

```diff
@@ -0,0 +1,13 @@
+package;
+
+import flixel.FlxGame;
+import openfl.display.Sprite;
+
+class Main extends Sprite
```

Comment:
All other demos use the Main.hx from the default template, might make sense to be consistent about that.